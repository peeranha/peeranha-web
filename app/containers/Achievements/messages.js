/*
 * Achievements Messages
 *
 * This contains all the text for the Achievements component.
 */

import { defineMessages } from 'react-intl';

export default defineMessages({
  QUESTION_ASKED: {
    title: {
      id: 'app.containers.Achievements.questionAsked.title',
    },
    multipleTitle: {
      id: 'app.containers.Achievements.questionAsked.multipleTitle',
    },
    description: {
      id: 'app.containers.Achievements.questionAsked.description',
    },
    bronzeTitle: {
      id: 'app.containers.Achievements.questionAsked.bronzeTitle',
    },
    bronzeDescription: {
      id: 'app.containers.Achievements.questionAsked.bronzeDescription',
    },
    silverTitle: {
      id: 'app.containers.Achievements.questionAsked.silverTitle',
    },
    silverDescription: {
      id: 'app.containers.Achievements.questionAsked.silverDescription',
    },
    goldTitle: {
      id: 'app.containers.Achievements.questionAsked.goldTitle',
    },
    goldDescription: {
      id: 'app.containers.Achievements.questionAsked.goldDescription',
    },
  },
  ANSWER_GIVEN: {
    title: {
      id: 'app.containers.Achievements.answerGiven.title',
    },
    multipleTitle: {
      id: 'app.containers.Achievements.answerGiven.multipleTitle',
    },
    description: {
      id: 'app.containers.Achievements.answerGiven.description',
    },
    bronzeTitle: {
      id: 'app.containers.Achievements.answerGiven.bronzeTitle',
    },
    bronzeDescription: {
      id: 'app.containers.Achievements.answerGiven.bronzeDescription',
    },
    silverTitle: {
      id: 'app.containers.Achievements.answerGiven.silverTitle',
    },
    silverDescription: {
      id: 'app.containers.Achievements.answerGiven.silverDescription',
    },
    goldTitle: {
      id: 'app.containers.Achievements.answerGiven.goldTitle',
    },
    goldDescription: {
      id: 'app.containers.Achievements.answerGiven.goldDescription',
    },
  },
  CORRECT_ANSWER: {
    title: {
      id: 'app.containers.Achievements.correctAnswer.title',
    },
    multipleTitle: {
      id: 'app.containers.Achievements.correctAnswer.multipleTitle',
    },
    description: {
      id: 'app.containers.Achievements.correctAnswer.description',
    },
    bronzeTitle: {
      id: 'app.containers.Achievements.correctAnswer.bronzeTitle',
    },
    bronzeDescription: {
      id: 'app.containers.Achievements.correctAnswer.bronzeDescription',
    },
    silverTitle: {
      id: 'app.containers.Achievements.correctAnswer.silverTitle',
    },
    silverDescription: {
      id: 'app.containers.Achievements.correctAnswer.silverDescription',
    },
    goldTitle: {
      id: 'app.containers.Achievements.correctAnswer.goldTitle',
    },
    goldDescription: {
      id: 'app.containers.Achievements.correctAnswer.goldDescription',
    },
  },
  FIRST_10K_REGISTERED: {
    title: {
      id: 'app.containers.Achievements.first10kregistered.title',
    },
    description: {
      id: 'app.containers.Achievements.first10kregistered.description',
    },
  },
  STRANGER: {
    title: {
      id: 'app.containers.Achievements.stanger.title',
    },
    description: {
      id: 'app.containers.Achievements.stanger.description',
    },
  },
  NEWBIE: {
    title: {
      id: 'app.containers.Achievements.newbie.title',
    },
    description: {
      id: 'app.containers.Achievements.newbie.description',
    },
  },
  JUNIOR: {
    title: {
      id: 'app.containers.Achievements.junior.title',
    },
    description: {
      id: 'app.containers.Achievements.junior.description',
    },
  },
  RESIDENT: {
    title: {
      id: 'app.containers.Achievements.resident.title',
    },
    description: {
      id: 'app.containers.Achievements.resident.description',
    },
  },
  SENIOR: {
    title: {
      id: 'app.containers.Achievements.senior.title',
    },
    description: {
      id: 'app.containers.Achievements.senior.description',
    },
  },
  HERO: {
    title: {
      id: 'app.containers.Achievements.hero.title',
    },
    description: {
      id: 'app.containers.Achievements.hero.description',
    },
  },
  SUPERHERO: {
    title: {
      id: 'app.containers.Achievements.superhero.title',
    },
    description: {
      id: 'app.containers.Achievements.superhero.description',
    },
  },
  ANSWER_15_MINUTES: {
    title: {
      id: 'app.containers.Achievements.answer15minutes.title',
    },
    description: {
      id: 'app.containers.Achievements.answer15minutes.description',
    },
    bronzeTitle: {
      id: 'app.containers.Achievements.answer15minutes.bronzeTitle',
    },
    bronzeDescription: {
      id: 'app.containers.Achievements.answer15minutes.bronzeDescription',
    },
    silverTitle: {
      id: 'app.containers.Achievements.answer15minutes.silverTitle',
    },
    silverDescription: {
      id: 'app.containers.Achievements.answer15minutes.silverDescription',
    },
    goldTitle: {
      id: 'app.containers.Achievements.answer15minutes.goldTitle',
    },
    goldDescription: {
      id: 'app.containers.Achievements.answer15minutes.goldDescription',
    },
  },
  FIRST_ANSWER: {
    title: {
      id: 'app.containers.Achievements.firstAnswer.title',
    },
    description: {
      id: 'app.containers.Achievements.firstAnswer.description',
    },
    bronzeTitle: {
      id: 'app.containers.Achievements.firstAnswer.bronzeTitle',
    },
    bronzeDescription: {
      id: 'app.containers.Achievements.firstAnswer.bronzeDescription',
    },
    silverTitle: {
      id: 'app.containers.Achievements.firstAnswer.silverTitle',
    },
    silverDescription: {
      id: 'app.containers.Achievements.firstAnswer.silverDescription',
    },
    goldTitle: {
      id: 'app.containers.Achievements.firstAnswer.goldTitle',
    },
    goldDescription: {
      id: 'app.containers.Achievements.firstAnswer.goldDescription',
    },
  },
});
