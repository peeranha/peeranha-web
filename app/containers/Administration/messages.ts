// @ts-ignore
import { defineMessages } from 'react-intl';

export default defineMessages({
  communityModerator: {
    id: 'app.containers.Administration.communityModerator',
  },
  communityAdministrator: {
    id: 'app.containers.Administration.communityAdministrator',
  },
  addRole: {
    id: 'app.containers.Administration.addRole',
  },
  chooseRole: {
    id: 'app.containers.Administration.chooseRole',
  },
  walletAddress: {
    id: 'app.common.walletAddress',
  },
  confirm: {
    id: 'app.containers.Administration.confirm',
  },
  cancel: {
    id: 'app.containers.Administration.cancel',
  },
  revoke: {
    id: 'app.containers.Administration.revoke',
  },
  wantToRevoke: {
    id: 'app.containers.Administration.wantToRevoke',
  },
});
