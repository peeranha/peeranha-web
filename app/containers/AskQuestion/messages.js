/*
 * AskQuestion Messages
 *
 * This contains all the text for the AskQuestion component.
 */

import { defineMessages } from 'react-intl';

export default defineMessages({
  title: {
    id: 'app.containers.AskQuestion.title',
    defaultMessage: 'Ask question',
  },
  description: {
    id: 'app.containers.AskQuestion.description',
    defaultMessage: 'Ask question | Description',
  },
  postQuestion: {
    id: 'app.containers.AskQuestion.postQuestion',
    defaultMessage: 'Post question',
  },
  notEnoughRating: {
    id: 'app.containers.AskQuestion.notEnoughRating',
    defaultMessage: 'To complete this action, your rating has to be more than',
  },
});
