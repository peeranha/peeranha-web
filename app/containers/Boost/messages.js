/*
 * Wallet Messages
 *
 * This contains all the text for the Wallet component.
 */

import { defineMessages } from 'react-intl';

export default defineMessages({
  title: {
    id: 'app.containers.Boost.title',
  },
  description: {
    id: 'app.containers.Boost.description',
  },
  currentPeriod: {
    id: 'app.containers.Boost.currentPeriod',
  },
  nextPeriod: {
    id: 'app.containers.Boost.nextPeriod',
  },
  maximumStake: {
    id: 'app.containers.Boost.maximumStake',
  },
  yourStake: {
    id: 'app.containers.Boost.yourStake',
  },
  yourBoost: {
    id: 'app.containers.Boost.yourBoost',
  },
  yourPredictedBoost: {
    id: 'app.containers.Boost.yourPredictedBoost',
  },
  formTitle: {
    id: 'app.containers.Boost.formTitle',
  },
  formSubmit: {
    id: 'app.containers.Boost.formSubmit',
  },
  formCancel: {
    id: 'app.containers.Boost.formCancel',
  },
  formBoostPrediction: {
    id: 'app.containers.Boost.formBoostPrediction',
  },
  formCurrentStake: {
    id: 'app.containers.Boost.formCurrentStake',
  },
  formTakeAStake: {
    id: 'app.containers.Boost.formTakeAStake',
  },
  notTokensToStake: {
    id: 'app.containers.Boost.notTokensToStake',
  },
});
