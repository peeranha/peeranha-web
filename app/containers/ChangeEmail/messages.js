import { defineMessages } from 'react-intl';

export default defineMessages({
  newEmail: {
    id: 'app.containers.ChangeEmail.newEmail',
  },
  confirmNewEmail: {
    id: 'app.containers.ChangeEmail.confirmNewEmail',
  },
  emailsDoNotMatch: {
    id: 'app.containers.ChangeEmail.emailsDoNotMatch',
  },
});
