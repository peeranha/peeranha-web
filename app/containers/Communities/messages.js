/*
 * Communities Messages
 *
 * This contains all the text for the Communities component.
 */

import { defineMessages } from 'react-intl';

export default defineMessages({
  title: {
    id: 'app.containers.Communities.title',
  },
  description: {
    id: 'app.containers.Communities.description',
  },
  tags: {
    id: 'app.containers.Communities.tags',
  },
  voting: {
    id: 'app.containers.Communities.voting',
  },
  suggestCommunity: {
    id: 'app.containers.Communities.suggestCommunity',
  },
  didntFindAnyInteresting: {
    id: 'app.containers.Communities.didntFindAnyInteresting',
  },
  suggestInterestingComm: {
    id: 'app.containers.Communities.suggestInterestingComm',
  },
});
