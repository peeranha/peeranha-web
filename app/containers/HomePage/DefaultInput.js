import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';
import { FormattedMessage } from 'react-intl';

import { LANDING_FONT } from 'style-constants';

const Wrapper = styled.div`
  * {
    font-family: ${LANDING_FONT};
    text-transform: none;
    letter-spacing: -0.9px;
  }

  h6 {
    margin: 9px 0;
    text-align: left !important;

    * {
      font-size: 14px !important;
      letter-spacing: -0.5px !important;
      color: #fc6655;
    }
  }

  .default-input {
    width: 100%;
    font-size: 18px;
    color: #282828;
    height: 48px;
    border: 1px solid #e6e6e6;
    border-radius: 3px;
    padding: 0 24px;
    background: #fff;

    ::placeholder {
      color: #9b9b9b;
    }

    :focus {
      border-color: #5c78d7;
    }
  }

  .default-input.default-input-error {
    border-color: #fc6655 !important;
  }
`;

const DefaultInput = /* istanbul ignore next */ ({
  input,
  disabled,
  meta: { touched, error, warning },
}) => (
  <Wrapper>
    <input
      {...input}
      type="text"
      className={`default-input ${touched &&
        (warning || error) &&
        'default-input-error'}`}
      placeholder="Your email address"
      disabled={disabled}
    />
    <h6 className="text-danger">
      {touched &&
        ((error && <FormattedMessage {...error} />) ||
          (warning && <FormattedMessage {...warning} />))}
    </h6>
  </Wrapper>
);

DefaultInput.propTypes = {
  input: PropTypes.object,
  meta: PropTypes.object,
  disabled: PropTypes.bool,
};

export default DefaultInput;
