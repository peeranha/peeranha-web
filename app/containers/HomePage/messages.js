/*
 * Landing Messages
 *
 * This contains all the text for the Landing component.
 */

import { defineMessages } from 'react-intl';

export default defineMessages({
  title: {
    id: 'app.containers.HomePage.title',
  },
  description: {
    id: 'app.containers.HomePage.description',
  },
  faqTitle: {
    id: 'app.containers.HomePage.faqTitle',
  },
  faqDescription: {
    id: 'app.containers.HomePage.title',
  },
  about: {
    id: 'app.containers.HomePage.about',
  },
  rewards: {
    id: 'app.containers.HomePage.rewards',
  },
  faq: {
    id: 'app.containers.HomePage.faq',
  },
  team: {
    id: 'app.containers.HomePage.team',
  },
  login: {
    id: 'app.containers.HomePage.login',
  },
  signUpFree: {
    id: 'app.containers.HomePage.signUpFree',
  },
  yourContributionsRewarded: {
    id: 'app.containers.HomePage.yourContributionsRewarded',
  },
  weAreDecentralized: {
    id: 'app.containers.HomePage.weAreDecentralized',
  },
  getStarted: {
    id: 'app.containers.HomePage.getStarted',
  },
  yourEmail: {
    id: 'app.containers.HomePage.yourEmail',
  },
  differentThan: {
    id: 'app.containers.HomePage.differentThan',
  },
  differentThan2: {
    id: 'app.containers.HomePage.differentThan2',
  },
  rewardingUser: {
    id: 'app.containers.HomePage.rewardingUser',
  },
  peopleSaveHours: {
    id: 'app.containers.HomePage.peopleSaveHours',
  },

  ownershipByUsers: {
    id: 'app.containers.HomePage.ownershipByUsers',
  },
  peeranhaDecentralized: {
    id: 'app.containers.HomePage.peeranhaDecentralized',
  },

  earlyAdopters: {
    id: 'app.containers.HomePage.earlyAdopters',
  },
  peeranhaPhilosophy: {
    id: 'app.containers.HomePage.peeranhaPhilosophy',
  },

  simpleAndAffordable: {
    id: 'app.containers.HomePage.simpleAndAffordable',
  },
  getHelp: {
    id: 'app.containers.HomePage.getHelp',
  },
  askQuestionAndGetHelp: {
    id: 'app.containers.HomePage.askQuestionAndGetHelp',
  },

  helpfulAndGenerous: {
    id: 'app.containers.HomePage.helpfulAndGenerous',
  },
  shareKnowledge: {
    id: 'app.containers.HomePage.shareKnowledge',
  },
  helpCommunity: {
    id: 'app.containers.HomePage.helpCommunity',
  },

  profitableAndModern: {
    id: 'app.containers.HomePage.profitableAndModern',
  },
  getPaidInCrypto: {
    id: 'app.containers.HomePage.getPaidInCrypto',
  },
  getPaidInCryptoFromWeekly: {
    id: 'app.containers.HomePage.getPaidInCryptoFromWeekly',
  },

  worthyAndPrestig: {
    id: 'app.containers.HomePage.worthyAndPrestig',
  },
  earnReputation: {
    id: 'app.containers.HomePage.earnReputation',
  },
  whetherYouAsk: {
    id: 'app.containers.HomePage.whetherYouAsk',
  },

  honorableAndFascinating: {
    id: 'app.containers.HomePage.honorableAndFascinating',
  },
  ruleThePlatform: {
    id: 'app.containers.HomePage.ruleThePlatform',
  },
  participateInModeration: {
    id: 'app.containers.HomePage.participateInModeration',
  },

  beTheFirst: {
    id: 'app.containers.HomePage.beTheFirst',
  },
  rewardsPool: {
    id: 'app.containers.HomePage.rewardsPool',
  },
  getReward: {
    id: 'app.containers.HomePage.getReward',
  },

  whatIsPeeranha: {
    id: 'app.containers.HomePage.whatIsPeeranha',
  },
  whatIsPeeranhaCollapsed: {
    id: 'app.containers.HomePage.whatIsPeeranhaCollapsed',
  },

  differentFromOtherSites: {
    id: 'app.containers.HomePage.differentFromOtherSites',
  },
  differentFromOtherSitesCollapsed: {
    id: 'app.containers.HomePage.differentFromOtherSitesCollapsed',
  },

  howItWorks: {
    id: 'app.containers.HomePage.howItWorks',
  },
  howItWorksCollapsed: {
    id: 'app.containers.HomePage.howItWorksCollapsed',
  },

  whatBlockchain: {
    id: 'app.containers.HomePage.whatBlockchain',
  },
  whatBlockchainCollapsed: {
    id: 'app.containers.HomePage.whatBlockchainCollapsed',
  },

  whatIsCryptocurrency: {
    id: 'app.containers.HomePage.whatIsCryptocurrency',
  },
  whatIsCryptocurrencyCollapsed: {
    id: 'app.containers.HomePage.whatIsCryptocurrencyCollapsed',
  },

  howEarn: {
    id: 'app.containers.HomePage.howEarn',
  },
  howEarnCollapsed: {
    id: 'app.containers.HomePage.howEarnCollapsed',
  },

  tokensReward: {
    id: 'app.containers.HomePage.tokensReward',
  },
  tokensRewardCollapsed: {
    id: 'app.containers.HomePage.tokensRewardCollapsed',
  },

  valueOfToken: {
    id: 'app.containers.HomePage.valueOfToken',
  },
  valueOfTokenCollapsed: {
    id: 'app.containers.HomePage.valueOfTokenCollapsed',
  },

  whitePaper: {
    id: 'app.containers.HomePage.whitePaper',
  },
  whitePaperCollapsed: {
    id: 'app.containers.HomePage.whitePaperCollapsed',
  },

  websiteLive: {
    id: 'app.containers.HomePage.websiteLive',
  },
  websiteLiveCollapsed: {
    id: 'app.containers.HomePage.websiteLiveCollapsed',
  },

  learnMore: {
    id: 'app.containers.HomePage.learnMore',
  },
  learnMoreCollapsed: {
    id: 'app.containers.HomePage.learnMoreCollapsed',
  },

  getMoreAnswers: {
    id: 'app.containers.HomePage.getMoreAnswers',
  },
  ourTeam: {
    id: 'app.containers.HomePage.ourTeam',
  },
  weAppreciate: {
    id: 'app.containers.HomePage.weAppreciate',
  },

  sergeyIlin: {
    id: 'app.containers.HomePage.sergeyIlin',
  },
  sergeyIlinDescription: {
    id: 'app.containers.HomePage.sergeyIlinDescription',
  },
  sergeyIlinRole: {
    id: 'app.containers.HomePage.sergeyIlinRole',
  },

  nikitaSyr: {
    id: 'app.containers.HomePage.nikitaSyr',
  },
  nikitaSyrRole: {
    id: 'app.containers.HomePage.nikitaSyrRole',
  },
  nikitaSyrDescription: {
    id: 'app.containers.HomePage.nikitaSyrDescription',
  },

  romRem: {
    id: 'app.containers.HomePage.romRem',
  },
  romRemRole: {
    id: 'app.containers.HomePage.romRemRole',
  },
  romRemDescription: {
    id: 'app.containers.HomePage.romRemDescription',
  },

  olgaFyodorova: {
    id: 'app.containers.HomePage.olgaFyodorova',
  },
  olgaFyodorovaRole: {
    id: 'app.containers.HomePage.olgaFyodorovaRole',
  },
  olgaFyodorovaDescription: {
    id: 'app.containers.HomePage.olgaFyodorovaDescription',
  },

  ulyanaPopova: {
    id: 'app.containers.HomePage.ulyanaPopova',
  },
  ulyanaPopovaRole: {
    id: 'app.containers.HomePage.ulyanaPopovaRole',
  },
  ulyanaPopovaDescription: {
    id: 'app.containers.HomePage.ulyanaPopovaDescription',
  },

  steveKreynin: {
    id: 'app.containers.HomePage.steveKreynin',
  },
  steveKreyninRole: {
    id: 'app.containers.HomePage.steveKreyninRole',
  },
  steveKreyninDescription: {
    id: 'app.containers.HomePage.steveKreyninDescription',
  },

  yourName: {
    id: 'app.containers.HomePage.yourName',
  },
  yourNameTip: {
    id: 'app.containers.HomePage.yourNameTip',
  },
  email: {
    id: 'app.containers.HomePage.email',
  },
  emailTip: {
    id: 'app.containers.HomePage.emailTip',
  },
  refCode: {
    id: 'app.containers.HomePage.refCode',
  },
  subject: {
    id: 'app.containers.HomePage.subject',
  },
  subjectTip: {
    id: 'app.containers.HomePage.subjectTip',
  },
  message: {
    id: 'app.containers.HomePage.message',
  },
  messageTip: {
    id: 'app.containers.HomePage.messageTip',
  },

  sendMessage: {
    id: 'app.containers.HomePage.sendMessage',
  },
  platformUnderDeveloping: {
    id: 'app.containers.HomePage.platformUnderDeveloping',
  },
  askQuestion: {
    id: 'app.containers.HomePage.askQuestion',
  },
  review: {
    id: 'app.containers.HomePage.review',
  },
  systemError: {
    id: 'app.containers.HomePage.systemError',
  },
  none: {
    id: 'app.containers.HomePage.none',
  },
  messageHasNotBeenSent: {
    id: 'app.containers.HomePage.messageHasNotBeenSent',
  },
  yourEmailIsRegistred: {
    id: 'app.containers.HomePage.yourEmailIsRegistred',
  },
});
