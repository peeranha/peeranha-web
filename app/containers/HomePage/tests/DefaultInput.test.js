import DefaultInput from '../DefaultInput';
import messages from '../messages';

const props = {
  input: {},
  meta: {
    touched: false,
    error: false,
    warning: false,
  },
  disabled: false,
  change: () => {},
  multiline: false,
};

describe('DefaultInput', () => {
  it('test1', () => {
    expect(DefaultInput(props)).toMatchSnapshot();
  });

  it('test2, touched, error - true', () => {
    props.meta.touched = true;
    props.meta.error = messages.title;

    expect(DefaultInput(props)).toMatchSnapshot();
  });
});
