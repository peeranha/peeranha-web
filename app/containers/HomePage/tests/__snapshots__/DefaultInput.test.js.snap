// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultInput test1 1`] = `
<DefaultInput__Wrapper>
  <input
    className="default-input false"
    disabled={false}
    placeholder="Email"
    type="text"
  />
  <h6
    className="text-danger"
  />
</DefaultInput__Wrapper>
`;

exports[`DefaultInput test2, touched, error - true 1`] = `
<DefaultInput__Wrapper>
  <input
    className="default-input default-input-error"
    disabled={false}
    placeholder="Email"
    type="text"
  />
  <h6
    className="text-danger"
  >
    <FormattedMessage
      id="app.containers.HomePage.title"
      values={Object {}}
    />
  </h6>
</DefaultInput__Wrapper>
`;
